.github/workflows/sync-docs-from-node.yml

```
@@ -25,879 +29,535 @@ on:
-        uses: actions/setup-node@v4
-        with:
-          node-version: '18'
-          cache: 'npm'
-
-      - name: Install dependencies
-        run: npm install
         
-      - name: Setup Python dependencies
+      - name: Extract version parameter
+        id: extract
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:6:28: Double quote to prevent globbing and word splitting

(shellcheck)